syntax = "proto3";
package monitor;

import "host_info.proto";
import "google/protobuf/timestamp.proto";

message MonitorInfo {
	google.protobuf.Timestamp collectTime = 1;
	HostInfo hostInfo = 2;
}

